src/packages/card/card.scss

```
@@ -27,8 +27,8 @@
 
     &-title {
       @include moreline-ellipsis();
-      line-height: 1.5;
-      font-size: 14px;
+      line-height: $line-height-2xl;
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)